sorc/chgres_cube.fd/search_util.f90

```diff
@@ -1,12 +1,12 @@
 !> @file
-!! @brief Replace undefined surface values.
-!!
-!! @author gayno NCEP/EMC
-!!
-!! Replace undefined values with a valid value.  This can
-!! happen for an isolated lake or island that is unresolved by
-!! the input grid.
-!!
+!> @brief Replace undefined surface values.
```

Comment:
Only the first comment in a doxygen block needs to be !>, the rest can be !!, so these need not be changed.

Intruducing blank lines changes the meening, so don't add the blank lines here.

Let's all use everyone's full name, so change ganyo to George Gayno everywhere it is used in a author tag, and the same for other names.

Reply:
I modified it again.

Reply:
Please take a look at this version, it contains update from a meeting took place today.